client/src/pages/WeeklyTable.js

```diff
@@ -108,14 +111,20 @@ const WeeklyTable = ({
 	return (
 		<div>
 			<div>
+				<button className={"inner"} onClick={setPreviousMonday}>
+					Prev week
+				</button>
+				<a href="#" class="previous">
+					&laquo;
+				</a>
 				<button className={"inner"} onClick={setThisMonday}>
 					This week
 				</button>
-				<button className={"inner"} onClick={setPreviousMonday}>
-					Previous week
-				</button>
+				<a href="#" class="next">
```

Comment:
Please use React semantic ui &lt;Button&gt; rather than a: (see the `import from 'React'` lines in ModalBookingScreen)